operate/client/e2e-playwright/tests/processInstanceMigration.mocks.ts

```diff
@@ -34,15 +34,19 @@ const setup = async () => {
   }
 
   return {
-    processV1Instances: await createInstances(
-      'orderProcessMigration',
-      deploymentsV1[0].process.version,
-      10,
-      {
-        key1: 'myFirstCorrelationKey',
-        key2: 'mySecondCorrelationKey',
-        key3: 'myThirdCorrelationkey',
-      },
+    processV1Instances: await Promise.all(
+      [...new Array(10)].map((_, index) =>
+        createSingleInstance(
+          'orderProcessMigration',
+          deploymentsV1[0]!.process.version,
+
+          {
+            key1: 'myFirstCorrelationKey',
+            key2: 'mySecondCorrelationKey',
+            key3: `myCorrelationKey${index}`,
+          },
+        ),
+      ),
```

Comment:
This change is needed to provide each process instance with a unique message correlation key.